mewarpx/mewarpx/emission.py

```diff
@@ -1186,6 +1182,108 @@ def get_normals(self, x, y, z):
         return normals
 
 
+class ZDiscEmitter(Emitter):
+
+    """This injects over an x-y disc rather than a rectangle."""
+    geoms = ['RZ']
+
+    def __init__(self, conductor, T, inner_emission_radius=None,
+                 outer_emission_radius=None, transverse_fac=1.0, **kwargs):
+        """Initialize an emitter for a disc (circular) cathode.
+
+        Arguments:
+            conductor (warp.Assembly): Conductor object, used to obtain work
+                function. Can later grab other variables from this conductor.
```

Comment:
```suggestion
                function and z coordinate/direction.
```
Also a similar edit to `ZPlaneEmitter`